vendor/bat-native-ads/src/bat/ads/internal/covariates/log_entries/notification_ad_event_unittest.cc

```diff
@@ -0,0 +1,86 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "bat/ads/internal/covariates/log_entries/notification_ad_event.h"
+
+#include <sstream>
+
+#include "bat/ads/confirmation_type.h"
+#include "bat/ads/internal/base/unittest/unittest_base.h"
+
+// npm run test -- brave_unit_tests --filter=BatAdsFederatedLogEntries*
+
+namespace ads {
+
+class BatAdsFederatedLogEntriesNotificationAdEventTest : public UnitTestBase {
```

Comment:
Unit test should have same name as class, i.e. `BatAdsNotificationAdEventCovariateLogEntryTest`